lib/hocon/impl/config_delayed_merge_object.rb

```diff
@@ -170,4 +170,8 @@ def hash
     # note that "origin" is deliberately NOT part of equality
     @stack.hash
   end
+
+  def render_to_sb(sb, indent, at_root, at_key, options)
```

Comment:
Same question here; not sure if this should be `render_value_to_sb`.  Looking at the code, I *think* the `render_to_sb` in `AbstractConfigValue` is supposed to be generic and re-used across all subclasses, and then at the end of it it calls `render_value_to_sb` which is the one that should be specific per subclass... but we probably need to take a quick look at the upstream signatures and validate that.